src/cli.rs

```diff
@@ -38,6 +38,25 @@ pub struct New {
     pub config: PathBuf,
 }
 
+#[derive(Args, Clone)]
+pub struct NewPage {
+    /// The URL slug of the page to create
+    pub slug: String,
+
+    /// The optional path beneath the pages directory to create the
+    /// page at.
+    #[arg(short, long, value_name = "SUBPATH")]
+    pub subpath: Option<PathBuf>,
+
+    /// The title of the post
+    #[arg(short, long)]
+    pub title: Option<String>,
+
+    /// The path of the gempost config file
+    #[arg(short, long, value_name = "PATH", default_value = "./gempost.yaml")]
+    pub config: PathBuf,
+}
```

Comment:
I like the addition of letting users have pages in nested directories with `--subpath`! But it got me thinking…

Running this:
```
gempost new-page --subpath ./projects my-gempost-fork
```

Feels less obvious/ergonomic than just:
```
touch ./pages/projects/my-gempost-fork.gmi
touch ./pages/projects/my-gempost-fork.yaml
```

Having `gempost new` makes sense because it does things like generate a post ID and set the `updated` and `published` times, but if we remove those pre-defined metadata fields for pages (see my comment about that below), and instead it's just creating empty files, then I think having a `gempost new-page` command makes a lot less sense.

Reply:
I would say a dedicated command is better, if two files need to be created, and if those files have required fields in them. Otherwise, the command might be unnecessary yes.